content/applications/inventory_and_mrp/inventory/management/products/usage.rst

```diff
@@ -1,72 +1,240 @@
+
+Under the :guilabel:`Inventory` tab, scroll down to the :guilabel:`Packaging` section, and click
+:guilabel:`Add a line`. Then, a pop-up window appears, in which the following information for each
+packaging should be entered.
+
+- :guilabel:`Packaging`: Name of packaging that appears on sales/purchase orders as a packaging
+  option for the product.
+- :guilabel:`Contained quantity`: Amount of product in the packaging.
+- :guilabel:`Barcode`: Identifier used with the *Barcode* app to trace the packaging of a product
+  during stock moves or pickings. Leave blank if not in use.
+- :guilabel:`Company`: Indicates the packaging is only available at the selected company. Leave
+  blank to make the packaging available across all companies.
+
+.. example::
+   Create a packaging type for 6 cans of the product, `Grape Soda`, by naming the
+   :guilabel:`Packaging` to `6-pack` and setting the :guilabel:`Contained quantity` to `6` in the
+   pop-up window that appears after clicking on :guilabel:`Add a line`.
+
+   .. image:: usage/create-product-packages.png
+      :align: center
+      :alt: Create 6-pack case for product.
+
+When all the necessary information has been entered, either click :guilabel:`Save & Close` to save
+the packaging and return to the product detail form, or :guilabel:`Save & New` to save the packaging
+and create another one in a fresh pop-up window.
+
+.. example::
+   View all product packagings in the :guilabel:`Inventory` tab of the product form.
+
+   .. image:: usage/display-product-packaging.png
+      :align: center
+      :alt: Show packaging and contained quantities, specified on the product page form in the
+            Inventory tab.
+
+View all packagings
+-------------------
+
+To view all packagings that have been created, go to :menuselection:`Inventory app --> Configuration
+--> Product Packagings`. Doing so reveals a separate page with a complete list of all packagings
+that have been created for all products. On this page, create new packagings by clicking
+:guilabel:`Create`.
+
+.. example::
+   Packagings for 2 different kinds of sodas is displayed on the :guilabel:`Product Packagings`
+   page. Each soda type is configured with three types of packagings.
+
+   .. image:: usage/packagings.png
+      :align: center
+      :alt: List of different packagings for products.
+
+Apply packagings
+----------------
+
+Then, when creating a sales order in the :menuselection:`Sales` app, specify the packagings that
+should be used for the product(s). The chosen packaging is displayed on the :abbr:`SO (Sales Order)`
+under the :guilabel:`Package` field.
+
+.. example::
+   18 cans of the product, `Grape Soda`, is packed using three 6-pack packagings.
+
+   .. image:: usage/packagings-sales-order.png
+      :align: center
+      :alt: Assign packagings on the Sales Order Line.
+
+.. tip::
+   Packaging can be used in conjunction with Odoo :ref:`Barcode <inventory/barcode/setup/software>`.
```

Comment:
this ref is *technically* correct however we're trying to keep it a little simpler, as follows
`module/sub-section/description`.

In this case it could be any of these: 
- `inventory/barcode/software`
- `inventory/barcode/setup`
- `inventory/barcode/config`